query-engine/query-structure/src/default_value.rs

```diff
@@ -282,8 +283,17 @@ impl ValueGeneratorFn {
     }
 
     #[cfg(feature = "default_generators")]
-    fn generate_uuid() -> PrismaValue {
-        PrismaValue::Uuid(uuid::Uuid::new_v4())
+    fn generate_uuid(version: u8) -> PrismaValue {
+        PrismaValue::Uuid(match version {
+            4 => uuid::Uuid::new_v4(),
+            7 => {
+                let now = chrono::Utc::now();
+                let timestamp =
+                    uuid::Timestamp::from_unix(uuid::NoContext, now.timestamp() as u64, now.timestamp_subsec_nanos());
+                uuid::Uuid::new_v7(timestamp)
```

Comment:
Is there any reason why you're generating the timestamp manually instead of using `uuid::Uuid::now_v7()`?

Reply:
Probably because I did not know of its existence, will change